runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -648,8 +652,13 @@ class INTERPRETER_CLASS
 #endif /* J9SW_NEEDS_JIT_2_INTERP_CALLEE_ARG_POP */
 			/* Set the flag indicating that the caller was the JIT */
 			_currentThread->jitStackFrameFlags = J9_SSF_JIT_NATIVE_TRANSITION_FRAME;
-			/* If a stop request has been posted, handle it instead of running the native */
+			if (isMethodDefaultConflictForMethodHandle) {
+				buildJITResolveFrame(REGISTER_ARGS);
```

Comment:
I still think this is wrong - do you have a simple testcase I can run (preferably on java8). I'll try to debug and see why my preferred solution doesn't work.